sdk/core/azure-core/inc/azure/core/operation_status.hpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
```

Comment:
missing file docs

Reply:
Good catch, its duplicated info for the one type in the file.  I forgot to add it but will.